components/menu/menu-item-styles.js

```diff
@@ -4,34 +4,28 @@ import { css } from 'lit';
 export const menuItemStyles = css`
 	:host {
 		background-color: var(--d2l-menu-background-color);
-		border: 1px solid transparent;
-		border-top-color: var(--d2l-menu-border-color);
+		border-top: 1px solid var(--d2l-menu-border-color);
 		box-sizing: border-box;
 		color: var(--d2l-menu-foreground-color);
 		cursor: pointer;
 		display: block;
 		font-size: 0.8rem;
 		margin-top: -1px;
-		outline: none;
 		width: 100%;
 	}
 
 	:host(:hover),
 	:host([first]:hover) {
 		background-color: var(--d2l-menu-background-color-hover);
-		border-bottom: 1px solid var(--d2l-menu-border-color-hover);
-		border-top: 1px solid var(--d2l-menu-border-color-hover);
 		color: var(--d2l-menu-foreground-color-hover);
-		z-index: 2;
 	}
 
-	/** separated because Safari <15.4 is having trouble parsing these */
```

Comment:
Isn't this comment still applicable for now?

Reply:
The only overlap between the two is color now... but I will add this back just in case someone thinking to combine them in the future.